README.md

```diff
@@ -1,6 +1,133 @@
 # github-actions-runner-aws
 
-## About
+## Recommended: AWS CodeBuild Runner Module
+
+**The `codebuild/` module is the recommended approach for self-hosted GitHub Actions runners on AWS.**
+
+### Why CodeBuild?
+
+- **Zero Maintenance**: No Docker images to maintain or update
+- **Serverless**: No ECS clusters or tasks to manage
+- **Cost-Effective**: ~40% savings vs GitHub-hosted runners, pay per build minute
+- **AWS Native**: Direct IAM role integration, no credential management
+- **Secure**: Ephemeral runners with no persistent state
+
+### Quick Start (Production-Ready with GitHub App)
+
+**Recommended for production:** Use GitHub App authentication for enhanced security and better rate limits.
+
+```hcl
+module "github_runner" {
+  source = "github.com/Enterprise-CMCS/mac-fc-github-actions-runner-aws//codebuild?ref=v1.0.0"
```

Comment:
We should update the version to be latest or higher tag number since v1.0.0 is already taken.